Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.yml

```diff
@@ -0,0 +1,162 @@
+category: Messaging
+commonfields:
+  id: Mail Listener v2
+  version: -1
+configuration:
+- display: Mail Server Hostname / IP address
+  name: MailServerURL
+  required: true
+  type: 0
+- display: Fetch incidents
```

Comment:
please place the all fetch related params together, and make sure they are below the credentials